apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -4,6 +4,8 @@
 {{#inline}}
 <span>
 {{/inline}}
+    <fieldset>
```

Comment:
Hmm, this is potentially going to cause issues with inline display, see https://github.com/PrairieLearn/PrairieLearn/issues/10538#issuecomment-2311096280. `<fieldset>` is a block-level element. Can we achieve the same thing with `<span role="group" aria-label="...">`? Maybe test that and see if it behaves similarly with a screen reader (I admit to not knowing exactly what a fieldset will be announced as).

Reply:
This seems to work as intended. The screen reader announces this way and the `<fieldset>` both as a "group" when focused.

Reply:
Great! Looks like this is resolved in 6f54747.